src/docs/sphinx/dev_guide/mfem_dev.rst

```diff
@@ -0,0 +1,51 @@
+.. ## Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+.. ## other Serac Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+======================
+MFEM Development Build
+======================
+
+Streamlining back-porting features to MFEM
+------------------------------------------
+
+Occasionally, Serac developers may want to back-port useful features back to MFEM. MFEM has two separate build systems (``make`` and `CMake``).
+The default ``make`` system that is suggested requires installing suggested packages. Fortunately, MFEM's `CMake` build system is more compatible with Serac's CMake-based build system.
+Since Serac provides tools to build MFEM's dependencies, one can streamline the back-porting process by re-using Serac's existing host-config files located in `<SERAC_ROOT_DIR>/host-configs`.
+
+MFEM's cmake build instructions suggest copying a `default.cmake` file to `user.cmake` and making adjustments to CMake variables. If we use Serac's host-config files, we need to only make a few modifications to `user.cmake`, and we can avoid rebuilding MFEM's dependencies manually.
```

Comment:
Do we not need to also set all the `MFEM_USE_*` variables?  I don't see any logic to do this but I may be missing something

Reply:
default.cmake has all of those variables defined in them.  By copying that file into user.cmake then making adjustments you don't need to set all of them on the command line.

This way does have several side effects, non-reproducability and non-uniform builds across teammates.  But if you are doing development work in MFEM thats probably acceptable.

Reply:
It might have been clearer if I left this further down where the instructions say to flip MFEM_USE_MPI - what I meant is that there are other options that are part of the default serac built configuration that should be flipped - right now I think these are MFEM_USE_SUPERLU, MFEM_USE_SUNDIALS, and MFEM_USE_NETCDF plus MFEM_USE_CUDA and MFEM_USE_AMGX with CUDA builds

Reply:
Is sundials always on by default or is it particular particular builds?